src/modules/pmc/README.md

```diff
@@ -79,16 +99,18 @@ The `executionState`, `executionSubState` and `executionSubStateDetails` propert
-| 3 (failed)     | 4 (deserializingPackages)     | package_name(s)          | Packages is not an array type, invalid array element                                            |
-| 3 (failed)     | 5 (modifyingSources)          | source_name              | Failed to delete or create source file                                                          |
-| 3 (failed)     | 4 (updatingPackagesLists)     | empty                    | Refreshing list of packages (by running `apt-get update` command) failed. A source repository may be unreachable or access may be unauthorized |
-| 3 (failed)     | 5 (installingPackages)        | package_name(s)          | Installation of package(s) failed because they (or any of their dependencies) weren't found in the source repositories |
-| 4 (timedOut)   | 4 (updatingPackagesLists)     | empty                    | Refreshing list of packages (by running `apt-get update` command) took more than 1 min         |
-| 4 (timedOut)   | 5 (installingPackages)        | package_name(s)          | Installing packages took more than 10 min                                                       |
\ No newline at end of file
+| 3 (Failed)     | 1 (DeserializingJsonPayload)  | empty                    | Payload too large, unabled to parse JSON payload, not specified PackageManagerConfiguration     |
+| 3 (Failed)     | 2 (DeserializingDesiredState) | empty                    | Invalid DesiredState payload, incorrect types specified, neither specified Sources nor Packages |
+| 3 (Failed)     | 3 (DeserializingGpgKeys)      | key_id                   | GpgKeys is not a map type, invalid map value                                                    |
+| 3 (Failed)     | 4 (DeserializingSources)      | source_id                | Sources is not a map type, invalid map value                                                    |
+| 3 (Failed)     | 5 (DeserializingPackages)     | package_name(s)          | Packages is not an array type, invalid array element                                            |
+| 3 (Failed)     | 6 (DownloadingGpgKeys)        | key_id                   | Provided URL is not accessible                                                                  |
+| 3 (Failed)     | 7 (ModifyingSources)          | source_id                | Failed to delete or create source file                                                          |
+| 3 (Failed)     | 8 (UpdatingPackagesLists)     | empty                    | Refreshing list of packages by running `apt-get update` failed. A source repository may be unreachable or access may be unauthorized. |
+| 3 (Failed)     | 9 (InstallingPackages)        | package_name(s)          | Installation of package(s) failed because they (or any of their dependencies) weren't found in the source repositories |
+| 4 (TimedOut)   | 8 (UpdatingPackagesLists)     | empty                    | Refreshing list of packages by running `apt-get update` took more than 10 min                   |
```

Comment:
would it make sense to use the longer timeout to be on the safe side?

Reply:
yeah, I think so, and it would be more understandable in the code, because only in these two places we will be passing our custom timeout and checking for the ETIME status.